packages/block-library/src/search/index.php

```diff
@@ -27,7 +27,7 @@ function render_block_core_search( $attributes ) {
 		);
 	} else {
 		$label_markup = sprintf(
-			'<label for="%s" class="wp-block-search__label screen-reader-text">%s</label>',
+			'<label for="%s" class="wp-block-search__label components-visually-hidden">%s</label>',
```

Comment:
There might be some issues using component class name in this way, since a component's class name isn't meant to be used outside of the component.

This html is also going to be output to a WordPress site's frontend, so would the components styles be loaded? Not sure, I'd need to test it. 😄 

It might be preferable for it to continue using the `screen-reader-text` global style or for the block to have its own visually hidden style.

It may also be an idea to split some of the changes that need more discussion into a separate PR so that they don't hold up the other changes.

Reply:
> It might be preferable for it to continue using the screen-reader-text global style or for the block to have its own visually hidden style.

Yes, correct. PHP part should be left as is.